app/page.js

```diff
@@ -1,8 +1,11 @@
 import Link from "next/link";
+import Navbar from "../components/pages/Navbar";
+import "../styles/scss/globals.scss";
 
 export default function Home() {
   return (
     <div className="homeContainer">
+      <Navbar />
       <div className="slogan">
         <h1 className="slogan_h1">Organize your work</h1>
         <h1 className="slogan_h1">and life, finally.</h1>
```

Comment:
Consider using the structure of "&lt;h1&gt;....&lt;br&gt;....&lt;/h1&gt;" or "&lt;h1&gt;&lt;p&gt;...&lt;/p&gt;&lt;p&gt;...&lt;/p&gt;&lt;/h1&gt;"

Reply:
How about       
```
<div>
     <Navbar />
</div>
```